### PR TITLE
Convert docs shortlinks to /docs subpath and added a new one for /docs/crawler

### PR DESCRIPTION
**Pull request by Toflar**
Required for https://github.com/contao/contao/pull/6503.

I know that we already have doc links on top level but I think we should start nesting them in `/docs` just like we do for `/tv` and `/camp` etc.

Keeping this as a draft, you can vote on your preferred solution here: https://github.com/contao/to.contao.org/discussions/81

### CODE REVIEW

**Review by fritzmg — APPROVED** [review]

**Review by netzarbeiter — APPROVED** [review]

File: redirects.yaml [hunk 1]
```diff
@@ -268,6 +235,43 @@ developers-meeting:
...
+    en: https://docs.contao.org/manual/en/installation/install-contao/#hosting-configuration
+    de: https://docs.contao.org/manual/de/installation/contao-installieren/#hosting-konfiguration
+
+docs/core-extensions:
+    en: https://docs.contao.org/manual/en/core-extensions/
+    de: https://docs.contao.org/manual/de/core-erweiterungen/
+
+docs/crawler:
+    en: https://docs.contao.org/manual/en/system/maintenance/
+    de: https://docs.contao.org/manual/de/system/systemwartung/
```
Comment (leofeyer) [inline redirects.yaml:257]:
```suggestion
    en: https://docs.contao.org/manual/en/system/maintenance/#crawler
    de: https://docs.contao.org/manual/de/system/systemwartung/#crawler
```

File: redirects.yaml [hunk 2]
```diff
@@ -268,6 +235,43 @@ developers-meeting:
...
+    de: https://docs.contao.org/manual/de/installation/systemvoraussetzungen/
+
+docs/faq:
+    en: https://docs.contao.org/manual/de/faq/
+    de: https://docs.contao.org/manual/en/faq/
+
+docs/mumble:
+    en: https://docs.contao.org/manual/en/guides/mumble/
+    de: https://docs.contao.org/manual/de/anleitungen/mumble/
+
```
Comment (leofeyer) [inline redirects.yaml:270]:
```suggestion
```

I don‘t think we need this anymore.

Comment (Toflar) [reply redirects.yaml:270]: Yeah I thought so too but it's still in the docs and I think we should remove them in a different PR. It's unrelated.

Comment (leofeyer) [reply redirects.yaml:270]: Please note that this only removes the **new** link, not the existing one. We have to keep the latter of course, but we do not need to create a new one.

Comment (Toflar) [reply redirects.yaml]: We can, but I cannot take your suggestion then because it would break the existing link. I've updated the PR.

**Review by leofeyer — APPROVED** [review]

**Pull request merged**